recipe/meta.yaml

```diff
@@ -1,31 +1,30 @@
-{% set version = "2.9.0" %}
+{% set version = "2.12.2" %}
 
 package:
   name: astroid
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/a/astroid/astroid-{{ version }}.tar.gz
-  sha256: 5939cf55de24b92bda00345d4d0659d01b3c7dafb5055165c330bc7c568ba273
+  sha256: 4675ef501edbbb143b3d9bb4c81d5f6338f08f960beed2ce41a03dc4cd20d777
 
 build:
   number: 0
-  # trigger: 1
-  skip: true  # [py<36]
+  skip: true  # [py<37]
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - python
     - pip
     - pytest-runner
-    - setuptools >=20.0
-    - wheel
+    - setuptools !=62.6
+    - wheel !=0.37.1
   run:
     - python
-    - setuptools >=20.0
+    - setuptools !=62.6
```

Comment:
!= (here) and ~= (from the upstream specs) are not the same operator. (In fact, there are almost direct opposites each other semantically speaking.)